README.md

```diff
@@ -1,73 +1,68 @@
 # Nova quickstart
 
-This is a simple example of a nov base Discord bot inspired by the discord.js structure.
+This document illustrates the process of creating a Discord Bot using the 
+Nova platform.
+The Bot interacts with Nova using a library following the popular 
+[Discord.js](https://discord.js.org) structure.
 
-## What you'll be doing
-
-This repo is a simple example of a bot developped using Nova. It's a simple bot that executes `ping` (ICMP Echo) requests using slash-commands.
```

Comment:
Can you keep the warning about ICMP requests ?
The bot allows user to ping arbitrary hosts, because of that, access to the bot must be kept private